worker/cli.py

```diff
@@ -82,6 +82,37 @@ def extract_pose_points(pose):
 
     return new_pose
 
+def get_first_frame_pose_prompts(input_file):
+    capture = cv2.VideoCapture(input_file)
+    frame_number = 0 # to note the 1st frame with a person is detected 
+    poses = None 
+    confs = None
+
+    person_class_id = [id for id, name in model.names.items() if name == "person"][0] # to get class id of "person"
+    
+    while True:
+        success, frame = capture.read()
+        if not success:
+            break
+
+        result = model.predict(source=frame, device='cpu', classes=[person_class_id], verbose=False)[0] # 1st frame
+
+        if ((result.keypoints is not None) and 
+            (result.keypoints.xy is not None) and 
+            (result.keypoints.conf is not None) and 
+            (len(result.keypoints.xy) > 0)):
+                poses = result.keypoints.xy.cpu().numpy().astype(int)
+                confs = result.keypoints.conf.cpu().numpy()
+                break
+        
+        frame_number += 1
+
+    capture.release()
+
+    if (poses is None) or (confs is None) or (poses.size == 0) or (confs.size == 0):
+        return [], [], -1 # in case no person was detected in the video
```

Comment:
Nitpicking: I'd prefer throwing an exception as it is more explicit that something went wrong and you can capture it below. Otherwise you have an implicit contract of returning -1 and then checking for -1